### PR TITLE
Display a marker on the wall when interacting with the mouse

### PR DESCRIPTION
**Pull request by rdumusc**

### CODE REVIEW

File: tide/master/MasterApplication.cpp [hunk 1]
```diff
@@ -73,6 +73,11 @@
 
 #include <stdexcept>
 
+namespace
+{
+const int MOUSE_MARKER_ID = 123456;
```
Comment (tribal-tec) [inline tide/master/MasterApplication.cpp]: hmm, magic?

Comment (rdumusc) [inline tide/master/MasterApplication.cpp]: yes, it can be any value, as long as it does not conflict with the touch point ids. How would you do it?

Comment (tribal-tec) [inline tide/master/MasterApplication.cpp]: if the touch point IDs start from 0, use max_int here then?

Comment (rdumusc) [inline tide/master/MasterApplication.cpp]: yes they do, OK for int_max

**Comment by tribal-tec** [comment]
+1

**Pull request merged**